surge/blueprints.py

```diff
@@ -0,0 +1,94 @@
+import re
+from surge.projects import Project
+from surge.errors import SurgeMissingAttributeError
+
+
+class Blueprint(Project):
```

Comment:
Might be good to validate that launch/pause/cancel etc. in Project aren't called for a blueprint, either in this class or in Project (perhaps in a follow-up PR) https://github.com/CivJ/surge-python/blob/3134a80007faad6b524a28bdc2cf9885df0a1136/surge/projects.py#L200

Reply:
This is of course validated on the server side though so maybe it doesn't matter

Reply:
Hi Tim! Yea I was a little worried about this. But since a Blueprint project can either be "just the template" or a real project that was derived from a template, it seems like we'd want the full project API available in the second case. For the first case, I am hoping the server will protect us (which may not be a good assumption).